pkg/logging/json_logger.go

```diff
@@ -29,6 +29,7 @@ import (
 	"time"
 
 	"github.com/containerd/containerd/runtime/v2/logging"
+	"github.com/containerd/errdefs"
```

Comment:
```suggestion
	"github.com/containerd/containerd/errdefs"
```
(Should be changed to github.com/containerd/errdefs later in another PR)

Reply:
Nice catch, thanks.